spec/spec.md

```diff
@@ -419,12 +428,16 @@ where:
 ```
 {
     “auth_token”: “<auth_token>”,
+    “pub_key”: [“<public_key>", ...]
```

Comment:
Should we pluralize this now?
```suggestion
    “pub_keys”: [“<public_key>", ...]
```

Reply:
None of the other terms (`transaction`, `message`) are pluralized in method call parameters. It's primarily intended for machine consumption, so I don't know that it matters all that much, but happy to change them all for consistency.

Reply:
Replaced with `accounts` instead